network/devices/link_linux_test.go

```diff
@@ -166,4 +166,73 @@ var _ = Describe("Link Management", func() {
 			Expect(names).To(ContainElement(name))
 		})
 	})
+
+	Describe("GetStatistics", func() {
+		BeforeEach(func() {
+			cmd, _ := gexec.Start(exec.Command("sh", "-c", "mount -n -t sysfs sysfs /sys ||true"), GinkgoWriter, GinkgoWriter)
+			Expect(cmd.Wait("2s").ExitCode()).To(Equal(0))
```

Comment:
this is a very minor thing, but you could use [`Eventually(cmd).Should(gexec.Exit(0))`](http://onsi.github.io/gomega/#gexec-testing-external-processes) here